pkgs/applications/editors/tiled/default.nix

```
@@ -29,7 +32,11 @@ stdenv.mkDerivation rec {
   };
 
   nativeBuildInputs = [ pkg-config qbs wrapQtAppsHook ];
-  buildInputs = [ qtEnv zlib libGL ];
+  buildInputs = [
+    qtEnv zlib libGL
+  ] ++ optionals zstdSupport [
```

```suggestion
  ] ++ lib.optionals zstdSupport [
```

and remove `inherit (lib) optionals;` above.